views/dataset/show_nav.templ

```diff
@@ -0,0 +1,45 @@
+package dataset
+
+import (
+	"github.com/ugent-library/biblio-backoffice/ctx"
+	"github.com/ugent-library/biblio-backoffice/models"
+	"github.com/ugent-library/biblio-backoffice/views"
+	"net/url"
+)
+
+type NavArgs struct {
+	ActiveSubNav string
+	SubNavs      []string
```

Comment:
I would just make this a package level var (see views/publication/sub_nav.templ)
ctx.Ctx now has a SubNav field that you can use, no need top pass it around, and there is a SetSubNav middleware

Reply:
Didn't know that!

Reply:
fresh of the press ;-)

Reply:
updated